### PR TITLE
Sparkbox Sprint 22

### PR DESCRIPTION
**Pull request by kaseybon**
# READY FOR REVIEW

## Summary
- feat(STN-443): Sprint 20 QA feedback (#665)
- fix(STN-477): Card subcontent has padding when nothing in it (#668)
- feat(STN-458): add theme setting to toggle site animation enhancements (#675)
- fix(STN-476): Footer block description configuration (#679)
- chore: backstop update (#672)
- feat(STN-427): added new color pairings to the colorful theme (#677)
- feat(STN-471): Add banner component (#673)
- fix(STN-507): Fix spacing regressions (#682)

### TASK LIST:
- [x] Add compiled CSS & JS
- [x] Ensure Code Climate passes against the `develop` branch. Link here: https://codeclimate.com/github/SU-HSDO/suhumsci/pull/642
- [x] Assign to Mike for review.

## Need Review By (Date)
8/11

## Urgency
high

## Steps to Test
1. `npm run test` should pass in the root folder
1. If you are interested in the testing steps for associated PRs, please take a look at the individual PR's description and "Steps to Test" section.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)

### CODE REVIEW

File: .circleci/config/behat.yml [hunk 1]
```diff
@@ -38,6 +38,7 @@ default:
         content: "div#main-content, .region-content"
         footer: "#footer"
         global_footer: "#footer__global-footer"
+        banner: ".paragraph--type--hs-banner"
```
Comment (pookmish) [inline .circleci/config/behat.yml]: That't not really a region. This shouldn't go here

File: docroot/themes/humsci/humsci_basic/humsci_basic.theme [hunk 2]
```diff
@@ -8,6 +8,31 @@
 use Drupal\Component\Utility\Html;
 use Drupal\node\Entity\Node;
 
+/**
+ * Implements hook_preprocess_block().
+ */
+function humsci_basic_preprocess_block(&$vars) {
+  // The following code block (lines 16 - 30) is an adaptation of the code found within the su_humsci theme preprocess block. `docroot/themes/humsci/su_humsci_theme/su_humsci_theme.theme`
+
```
Comment (pookmish) [inline docroot/themes/humsci/humsci_basic/humsci_basic.theme]: No empty lines after a comment. also this comment line is too long. Code climate should have picked that up. This comment could go below the `Implements` in the docblock.

**Review by pookmish — CHANGES_REQUESTED** [review]
Just a couple small changes requested

**Comment by kaseybon** [comment]
@pookmish should be all set now

**Review by pookmish — APPROVED** [review]

**Pull request merged**